wordpress-mobile/WordPress-Aztec-iOS
# Adds initial copy / paste support.

Related to #75.

Adds initial copy / paste support.  This PR is a bit large but at least half of the added lines are unit tests.

**Known limitations:**

When copy-pasting styles, there's some HTML node duplication (example: `<b>hello</b><b> there</b>`).  This'll be addressed in an upcoming PR, so please ignore for now.

**How to test:**

Run the unit tests... then:

1. Launch the example app.
2. Copy styled text.
3. Paste it into the editor.
4. Switch to HTML mode and make sure the tags are there (the supported ones are: bold, strike, underline, italic).
